README.md

```diff
@@ -644,6 +644,9 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
 - `booleans` -- various optimizations for boolean context, for example `!!a
   ? b : c → a ? b : c`
 
+- `typeofs` -- `typeof foo == "undefined"` → `foo === void 0`
+  Note: Internet Explorer 10 or below has known issue with special values of `foo`.
```

Comment:
Note: recommend setting this value to `false` for IE10 and earlier versions due to known issues.